app/src/androidTest/java/com/example/partyPlayer/InstrumentedTest.kt

```diff
@@ -1,4 +1,4 @@
-package com.example.party_player
+package com.example.partyPlayer
```

Comment:
Changed the package names by removing underscores to comply with Kotlin naming conventions stated here: https://kotlinlang.org/docs/reference/coding-conventions.html

Reply:
While they are two words, package names are usually all lowercase.  Consider making this all lowercase.